app/src/screens/SignIn/index.tsx

```diff
@@ -61,10 +64,36 @@ const styles = StyleSheet.create({
     height: 20,
     marginLeft: 4,
   },
+  errorWarning: {
+    height: 20,
+    width: 20,
+  },
+  errorContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    width: '90%',
+  },
+  errorText: {
+    paddingLeft: 4,
+    lineHeight: 20,
+    fontSize: 14,
+    color: '#D61F1F',
+  },
 });
 
 const SignIn: React.FC = () => {
   const dispatch = useDispatch<Dispatch>();
+  const accessTokenState = useSelector(
+    (state: State) => state.voice.accessToken,
+  );
+  const [errorMessage, setErrorMessage] = React.useState<string>('');
+
+  useEffect(() => {
+    if (accessTokenState.status === 'rejected') {
+      setErrorMessage(accessTokenState.error.message);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [accessTokenState.status]);
```

Comment:
We can move all this logic into the selector callback itself on line `87`. That way we don't have more state than we need and we don't need a potential side-effect from this `useEffect` hook. Also, if we need to use a linting rule to prevent dependency checks, we're probably missing something important.

Reply:
I have an issue when I move this to `useSelector` I get a "too many re-renders" error